products/pdt/cd/base/pv.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: PersistentVolume
```

Comment:
I think if you have a `PersistentVolumeClaim` already this should be unnecessary? 

https://kubernetes.io/docs/concepts/storage/persistent-volumes/#reserving-a-persistentvolume

> The control plane can [bind PersistentVolumeClaims to matching PersistentVolumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#binding) in the cluster. However, if you want a PVC to bind to a specific PV, you need to pre-bind them.

(I don't think we need to pre-bind here?)

Reply:
Ah makes sense. In that case I will just remove PV